app/main.py

```diff
@@ -3,7 +3,7 @@ def count_occurrences(phrase: str, letter: str) -> int:
     Implement count_occurrences function:
 
     It takes a phrase and a letter and calculates the number of times
-    the letter appears in the phrase. The function is case insensitive.
+    the letter appears in the phrase. The function is case-insensitive.
```

Comment:
You should skip changing it here)

Reply:
Okay, Pycharm was yelling at me for that, so...